examples/semantic_segmentation/index.html

```diff
@@ -418,6 +418,9 @@ <h5 class='modal-title' id='modaltitle'>Subgraphs Summary</h5>
   <script src='../util/tflite/schema/schema_generated.js'></script>
   <script src='../util/tflite/TfLiteModelUtils.js'></script>
   <script src='../util/tflite/TFliteModelImporter.js'></script>
+  <script src='../util/onnx/onnx.js'></script>
+  <script src='../util/onnx/OnnxModelUtils.js'></script>
+  <script src='../util/onnx/OnnxModelImporter.js'></script>
```

Comment:
Thanks, if we don't need onnx, it's good to remove them.

Reply:
> Thanks, if we don't need onnx, it's good to remove them.

Iteratorlayer tool will crash without these lines, so I added them.